src/parsers/directive.parser.ts

```diff
@@ -17,24 +17,24 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		let results: string[] = [];
 
 		template = this._normalizeTemplateAttributes(template);
-		$(template)
```

Comment:
Can we keep this code I just pushed?

I think its cleaner for a number of reasons.

- No magic number for text nodes
- Only uses arrow functions
- Doesn't jump between element, children and parent

... and should be faster since it only loops element's textnodes if no attribute key was found.